Rubberduck.Core/UI/CodeExplorer/Commands/RemoveCommand.cs

```diff
@@ -31,33 +45,59 @@ protected override void OnExecute(object parameter)
                 return;
             }
 
-            var qualifiedModuleName = node.Declaration.QualifiedName.QualifiedModuleName;
-            var message = string.Format(CodeExplorerUI.ExportBeforeRemove_Prompt, node.Name);
+            RemoveComponent(node.Declaration.QualifiedModuleName);
```

Comment:
I don't think this can ever be null - `QualifiedModuleName` is a struct. The last class in the chain (`Declaration`) is already null-checked in the guard clause.

Reply:
You are right. I overlooked that check.